posthog/demo/matrix/manager.py

```diff
@@ -194,14 +194,14 @@ def _copy_analytics_data_from_master_team(self, target_team: Team):
         sync_execute(COPY_PERSON_DISTINCT_ID2S_BETWEEN_TEAMS, copy_params)
         sync_execute(COPY_EVENTS_BETWEEN_TEAMS, copy_params)
         sync_execute(COPY_GROUPS_BETWEEN_TEAMS, copy_params)
+        GroupTypeMapping.objects.filter(team_id=target_team.pk).delete()
```

Comment:
Hm, I don't quite see how this change would affect persons or dashboards. I also don't know of any way for a freshly-created `target_team` to have any GroupTypeMapping data, so that's suspicious. Can the problem being solved be recreated with a test?

Reply:
Discussed in Slack: There isn't really indication this will help, but it's good for data integrity anyway, so let's see. The next steps for debugging involve adding logs.